auto_submit/lib/service/config.dart

```diff
@@ -75,6 +76,9 @@ class Config {
         'DartDevtoolWorkflowBot',
       };
 
+  //TODO change this back to autosubmit[bot] after testing is complete.
+  String get autosubmitBot => 'auto-submit[bot]';
```

Comment:
Is the `testing` complete? This is `auto-submit[bot]` already?

Reply:
Yes it is complete. I just created the pull request in the .github repo to add this to the official config. I will clean this up.

Reply:
Removed.